dotCMS/src/main/java/com/dotcms/publishing/PublisherAPIImpl.java

```diff
@@ -167,6 +169,33 @@ final public PublishStatus publish ( PublisherConfig config, PublishStatus statu
         return status;
     }
 
+    private void addBundleXMLIntoBundle(final PublisherConfig config, final BundleOutput output) {
+        if (config.isStatic()) {
+            //If static we just want to save the things that we need,
+            // at this point only the id, static and operation.
+            PublisherConfig pcClone = new PublisherConfig();
```

Comment:
`pcClone` can be declared final

Reply:
done https://github.com/dotCMS/core/pull/20964/commits/112eb77d7bac9805df209b40ddd5e547901f6842#diff-bd637ff1eb9c8bc20a211e7cd15332f0f1253b19abb2c88074e21347ef539fc4R164